Ports/alpine/package.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=alpine
+version=2.26
+useconfigure=true
+configopts=(
+    "--disable-static" \
+    "--enable-shared" \
+)
+use_fresh_config_sub=true
+use_fresh_config_guess=true
+files="https://alpineapp.email/alpine/release/src/alpine-2.26.tar.xz alpine-${version}.tar.xz c0779c2be6c47d30554854a3e14ef5e36539502b331068851329275898a9baba"
+auth_type=sha256
+depends=("openssl" "ncurses" "ca-certificates")
```

Comment:
Break into one per line for nicer future diffs

Reply:
Good tip, done.